symfony/client/src/orangehrmAdminPlugin/pages/systemUser/EditSystemUser.vue

```diff
@@ -155,17 +155,17 @@ export default {
           shouldNotExceedCharLength(40),
           promiseDebounce(this.validateUserName, 500),
         ],
-        role: [v => (!!v && v.length != 0) || 'Required'],
+        role: [required],
```

Comment:
@https://github.com/Super-Chama I've changed this validation into the required validation in rules.ts and based on testing on the view it gave the error message when kept empty. Hope that's ok

Reply:
Yeah. fine we have to track these changes. may be screen wise